src/services/agent-service.ts

```diff
@@ -68,20 +69,23 @@ export default class AgentService {
 
     if (!this.snapshotService) { return response.json({success: false}) }
 
+    const snapshotConfiguration = (configuration().snapshot || {}) as SnapshotConfiguration
+    const snapshotOptions: SnapshotOptions = {
+      widths: request.body.widths || snapshotConfiguration.widths,
+      enableJavaScript: request.body.enableJavaScript,
```

Comment:
@djones I notice that `SnapshotConfiguration` does _not_ have an `enableJavaScript` field. Should it?

Reply:
It's ok to add it if it makes things easier. Generally speaking we don't want to encourage people to `enableJavaScript` though so that's why it was left off. But I don't feel super strongly about that.

Reply:
If we prefer people not to use it, then let's leave it off until a time when we actually do need it.